index.js

```diff
@@ -8,29 +8,32 @@ module.exports = bel
 module.exports.update = function (fromNode, toNode, opts) {
   if (!opts) opts = {}
   if (opts.events !== false) {
-    if (!opts.onBeforeElUpdated) opts.onBeforeElUpdated = copier
+    if (!opts.onBeforeElUpdated) opts.onBeforeElUpdated = currier(opts.onBeforeElUpdated, opts)
```

Comment:
This PR is still using the currier only if `onBeforeElUpdated` is -not- provided by the user.

Expected:
```javascript
opts.onBeforeElUpdated = (opts.onBeforeElUpdated) ?
  currier(opts.onBeforeElUpdated, opts) : 
  copier;
```

The function `currier` is also immediately calling the `update` function on first assignment, instead of returning a function that calls `update` then `copier` (provided the `update` function has not returned false).

Expected:
```javascript
function currier(update, opts){
  return function (f, t){
    const userUpdatePassed = update(f, t);
    if (userUpdatePassed) return copier(f, t);
   return false; // or just return if morphdom doesn't care about undefined/false distinction
  }
}
function copier(f, t){}
```
Thanks!